### PR TITLE
Garnett: feature showcase

### PR DESCRIPTION
**Pull request by AWare**
## What does this change?
implements @superfrank's feature showcase design

## What is the value of this and can you measure success?
garnett changes, no

## Does this affect other platforms - Amp, Apps, etc?
no
## Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

no

### CODE REVIEW

**Comment by AWare** [comment]
there's a bad case of the magic numbers going on here, working on it.

**Comment by AWare** [comment]
I need to go through this with Frank and make sure I've got the positioning and margins right.

I've updated the trello with the crops this will will affect on the main image.
https://trello.com/c/dk7Kv2pg/51-2feature-showcase-article-development

File: static/src/stylesheets/_vars.scss [hunk 1]
```diff
@@ -59,7 +59,7 @@ $mq-breakpoints: (
 // Show the current breakpoint in the top right corner of the viewport:
 // see https://github.com/guardian/sass-mq#seeing-the-currently-active-breakpoint
 // Uncomment to activate during development
-// $mq-show-breakpoints: (mobile, mobileLandscape, tablet, desktop, leftCol, wide);
```
Comment (NataliaLKB) [inline static/src/stylesheets/_vars.scss]: This should still be commented right?

File: static/src/stylesheets/module/content-garnett/_types.scss [hunk 2]
```diff
@@ -1077,13 +1077,107 @@ $quote-mark: 35px;
 }
 // *************** Feature showcase overrides ***************
 
+// Articles can have the feature tone applied, but it's not always primary tone
 .has-feature-showcase-element {
+    .tonal__header > .u-cf {
```
Comment (NataliaLKB) [inline static/src/stylesheets/module/content-garnett/_types.scss]: We try not to rely on `u-cf` for styling, as it is just a helper class

File: static/src/stylesheets/module/content-garnett/_types.scss [hunk 3]
```diff
@@ -1077,13 +1077,107 @@ $quote-mark: 35px;
...
+        .content__labels--not-immersive {
+            display: none;
+        }
+        //@include mq($until: leftCol){
```
Comment (NataliaLKB) [inline static/src/stylesheets/module/content-garnett/_types.scss]: Should this be commented, or deleted?

File: static/src/stylesheets/module/content-garnett/_types.scss [hunk 4]
```diff
@@ -1077,13 +1077,107 @@ $quote-mark: 35px;
...
+            width: gs-span(4);
+        }
+        @include mq(tablet) {
+            width: gs-span(6);
+        }
+        @include mq(leftCol) {
+            background: #ffffff;
+            margin-top: 0;
+            position: absolute;
+            top: 10px;
```
Comment (NataliaLKB) [inline static/src/stylesheets/module/content-garnett/_types.scss]: Any chance you could use `gs-baseline`? See here for grid: https://github.com/guardian/guss-grid-system

File: static/src/stylesheets/module/_layout-hints.garnett.scss [hunk 5]
```diff
@@ -0,0 +1,266 @@
...
+
+    @include mq(tablet) {
+        width: gs-span(2);
+
+        &.img--landscape {
+            margin-right: $gs-gutter; // repeat to override negative margin on img--landscape
+        }
+    }
+
+    @include mq(leftCol) {
```
Comment (regiskuckaertz) [inline static/src/stylesheets/module/_layout-hints.garnett.scss]: do you need that extra `mq` if it is already there in `deport-left`?

Comment (regiskuckaertz) [reply static/src/stylesheets/module/_layout-hints.garnett.scss]: (not that it matters much, should be compiled away)

File: static/src/stylesheets/_vars.scss (hunk 1 above)
Comment (AWare) [reply static/src/stylesheets/_vars.scss]: It should be.

**Comment by AWare** [comment]
change of design reqs

**Comment by AWare** [comment]
Design changes needed.

**Pull request closed without merging**